pkg/helm/cmd.go

```diff
@@ -83,14 +84,14 @@ func installOrUpgrade(ctx context.Context, getter genericclioptions.RESTClientGe
 		if err != nil {
 			return nil, err
 		}
-		logger.Info("install completed", ReleaseLog(rel)...)
+		logger.Info(fmt.Sprintf("helm install completed dryRun:%t", dryRun), ReleaseLog(rel)...)
```

Comment:
**Suggestion:** Use structured logging instead of `fmt.Sprintf` for better log readability and performance. The `logr` package supports structured logging.
```suggestion
  logger.Info("helm install completed", "dryRun", dryRun, "release", ReleaseLog(rel)...)
```